### PR TITLE
V3

### PR DESCRIPTION
**Pull request by kirjs**
Limit to React@~0.13
Copy all modules in modules folder
Test All 3 bundles
Add Highmaps
Add Highmaps demo
Generate contents for demos
Clean up build scripts
internal paths clean up

### CODE REVIEW

**Comment by kirjs** [comment]
#41 #44 @aldendaniels @Kureev

**Comment by jasongonzales23** [comment]
Is the plan to also promote the demos to React 0.14?

**Comment by kirjs** [comment]
Yes

File: src/chartsFactory.jsx [hunk 1]
```diff
@@ -40,6 +45,7 @@ module.exports = function (Highcharts, chartType) {
     },
 
     componentDidMount: function () {
+      debugger
```
Comment (Kureev) [inline src/chartsFactory.jsx]: remove `debugger`

**Comment by Kureev** [comment]
:+1:

**Pull request merged**